apps/meteor/ee/client/voip/modal/DialPad/DialPadModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Field, Modal, IconButton } from '@rocket.chat/fuselage';
+import React, { ReactElement } from 'react';
+import MaskedInput from 'react-text-mask';
```

Comment:
To be honest, for the level of masking we are delivering on this first release, this lib seems overkill. 🤔